skimage/metrics/_structural_similarity.py

```diff
@@ -194,11 +194,22 @@ def structural_similarity(im1, im2,
         raise ValueError('Window size must be odd.')
 
     if data_range is None:
+        if (np.issubdtype(im1.dtype, np.floating) or
+            np.issubdtype(im2.dtype, np.floating)):
+            raise ValueError(
+                'Since image dtype is floating point, the data_range '
+                'parameter must be specified. Carefully read the '
+                'documentation (including the note). It is recommended '
+                'to always specify the data_range anyway.')
```

Comment:
```suggestion
                'Since image dtype is floating point, you must specify '
                'the data_range argument. Please read the documentation '
                'carefully (including the note). It is recommended that '
                'you specify the data_range anyway.')
```
(always/anyway were somewhat redundant)